mappings/net/minecraft/client/particle/VibrationParticle.mapping

```diff
@@ -1,5 +1,13 @@
 CLASS net/minecraft/class_5737 net/minecraft/client/particle/VibrationParticle
 	FIELD field_28249 vibration Lnet/minecraft/class_5720;
+	METHOD <init> (Lnet/minecraft/class_638;Lnet/minecraft/class_5720;I)V
+		ARG 1 world
+		ARG 2 vibration
+		ARG 3 maxAge
+	METHOD method_33078 (Lnet/minecraft/class_4588;Lnet/minecraft/class_4184;FLjava/util/function/Consumer;)V
```

Comment:
This seems to be just drawing the vibration particle. In addition, the consumer can probably be named `transforms`